server/controllers/stock/reports/index.js

```diff
@@ -685,31 +721,56 @@ function stockInventoriesReport(req, res, next) {
   let options = {};
   let display = {};
   let hasFilter = false;
+  let report;
 
   const data = {};
-  let report;
-  const optionReport = _.extend(req.query, { filename : 'TREE.STOCK_INVENTORY', orientation : 'landscape' });
+  const bundle = {};
+
+  const optionReport = _.extend(req.query, {
+    filename : 'TREE.STOCK_INVENTORY',
+    orientation : 'landscape',
+    footerRight : '[page] / [toPage]',
+    footerFontSize : '8',
+  });
 
   // set up the report with report manager
   try {
     if (req.query.identifiers && req.query.display) {
       options = JSON.parse(req.query.identifiers);
       display = JSON.parse(req.query.display);
       hasFilter = Object.keys(display).length > 0;
+    } else if (req.query.params) {
```

Comment:
Why do you have a separate `req.query.params`?  Can we not put this all in `req.query`?